main/src/library/InputStream.flix

```diff
@@ -0,0 +1,89 @@
+namespace InputStream {
+    ///
+    /// Wraps the `java.io.InputStream.available()` method that returns the number of available bytes
```

Comment:
Replace by a one sentence description that explains what the function does. (Don't copy from Javadoc, since that is copyrighted).